src/style.css

```diff
@@ -1 +1,96 @@
-/* add styles here */
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+:root {
+  --font-color: #3d4e61;
+}
+
+.page {
+  font-family: Avenir, sans-serif;
+}
+
+.form {
+  justify-content: center;
+  margin: 20px auto;
```

Comment:
Bad idea to use margins here.  Basically, you want to avoid `margin collapse` whenever you can to. 
Why don't you create a flex container for these forms and add `gap` to it?